stock_inventory/tests/test_stock_inventory.py

```diff
@@ -0,0 +1,593 @@
+# Copyright 2022 ForgeFlow S.L
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo.exceptions import UserError, ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestStockInventory(TransactionCase):
```

Comment:
```suggestion
class TestStockInventory(BaseCommon):
```

This is not really necessary, but it will help with performance.

Reply:
Done